jf_agent/git/github_gql_adapter.py

```diff
@@ -316,9 +318,9 @@ def get_pull_requests(
                         f'getting PRs for repo {nrm_repo.name} ({nrm_repo.id}). Skipping...',
                         log_as_exception=True,
                     )
-        return nrm_prs
 
-    print('✓')
+        logger.info('Done downloading github PRs!')
```

Comment:
Q: Was the `print('✓')` in the old version an error? Looks like it's on the same line as the function definition.

Reply:
The old version was an indentation bug. I just confirmed again... the `Done downloading github PRs!` is at the same indentation as `downloading github prs...`

This graphql adapter is used by only a few clients so far, so the escaped bug impact was pretty small (and limited to only stdout UX)